pdk/1.0/modules/ROOT/pages/policies-pdk-create-project.adoc

```diff
@@ -92,9 +84,9 @@ After running the setup command, the policy's root directory contains the follow
 There are two separate `target` directories:
 
 * `definition/target/`: Contains the policy asset definition published to Exchange.
-* `target/`: Contains the files for the Policy asset implementation and the output of the Rust compilation process.
+* `target/`: Contains the files for the policy's Exchange asset implementation and the output of the project build
 
-Both `target` directories and their content are included in the `.gitignore` file. You must compile the targets on each individual devices. For example, if you are developing a policy as part of a team, your team can publish the policy and it's source code to a remote repository, but each team member must build the target on their device for local testing.
+Both target directories and their content are included in the .gitignore file. You must compile the targets on each individual devices using the `make build` command. For example, if you develop a policy as part of the team, your team can publish the policy and its source code to a remote repository, but each team member must build the target on their device to work with the policy.
```

Comment:
I'm curious why you removed the formatting from .gitignore? According to our standards, files names should be in mono font: https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#:~:text=a%20faull%20filename.-,font%20choices,-Follow%20these%20rules